superset-frontend/src/explore/components/controls/DateFilterControl/DateFilterLabel.tsx

```diff
@@ -373,7 +373,7 @@ export default function DateFilterLabel(props: DateFilterControlProps) {
       >
         <Tooltip placement="top" title={tooltipTitle}>
           <Label className="pointer" data-test="time-range-trigger">
-            {actualTimeRange}
+          {t(`${actualTimeRange}`)}
```

Comment:
I don't know what strings would be expected here, but this sounds highly dynamic and unlikely to be in the translation files. Correct me if I'm mistaken.